actors/builtin/miner/miner_actor.go

```diff
@@ -1441,27 +1445,23 @@ func scheduleReplaceSectorsExpiration(rt Runtime, st *State, store adt.Store, re
 	// That's a very brittle constraint, and would be much better with two-phase termination (where we could
 	// deduct power immediately).
 	// See https://github.com/filecoin-project/specs-actors/issues/535
-	replaceSectorNumbers := make([]uint64, len(replaceSectors))
 	deadlineInfo := st.DeadlineInfo(rt.CurrEpoch())
 	newExpiration := deadlineInfo.PeriodEnd()
-	for i, sector := range replaceSectors {
-		// Note: State updates could be minimized by batching these by existing expiration epoch.
-		if err := st.RemoveSectorExpirations(store, sector.Expiration, uint64(sector.SectorNumber)); err != nil {
-			return fmt.Errorf("failed to remove sector expiration %v at %v: %w",
-				sector.SectorNumber, sector.Expiration, err)
-		}
+
+	if err := st.RemoveSectorExpirations(store, replaceSectors...); err != nil {
+		return err
```

Comment:
I have moderate preference for wrapping these errors so that we or the node implementation have more information when debugging whatever corruption might cause it.

Reply:
Done.